ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/computation/StatementExecutorTest.java

```diff
@@ -114,11 +122,12 @@ public void handleCSAS_CTASStatement() throws Exception {
 
     Map<CommandId, CommandStatus> statusStore = statementExecutor.getStatuses();
     Assert.assertNotNull(statusStore);
+    Assert.assertEquals(6, statusStore.size());
```

Comment:
you can create a `Map<CommandId, CommandStatus>` call it expected and then do:
`assertThat(statusStore, equalTo(expected))`

Reply:
The problem is that CommandStatus has two fields, status and message. We only case about the status in this case so if creating CommandStatus with specific message, especially exception message for ERROR case may be much more complex.